src/org/omegat/filters2/po/PoFilter.java

```diff
@@ -306,51 +317,11 @@ public String getFileFormatName() {
 
     @Override
     public Instance[] getDefaultInstances() {
-        return new Instance[]
-        { new Instance("*.po", StandardCharsets.UTF_8.name(), StandardCharsets.UTF_8.name()),
+        return new Instance[] {
+                new Instance("*.po", StandardCharsets.UTF_8.name(), StandardCharsets.UTF_8.name()),
                 new Instance("*.pot", StandardCharsets.UTF_8.name(), StandardCharsets.UTF_8.name()) };
     }
 
-    /**
-     * return BufferdReader for PoFile reading.
-     * @param inFile
-     *            The source file.
-     * @param inEncoding
-     *            Encoding of the input file, if the filter supports it. Otherwise null.
-     * @return
-     * @throws IOException
-     */
-    @Override
-    protected BufferedReader createReader(File inFile, String inEncoding) throws IOException {
-        BOMInputStream bomInputStream = new BOMInputStream(Files.newInputStream(inFile.toPath()),
-                ByteOrderMark.UTF_8, ByteOrderMark.UTF_16LE);
-        ByteOrderMark bom = bomInputStream.getBOM();
-        String charset;
```

Comment:
Now we have `AbstractFilter#createReader` which as same functionality with one here, so we can remove it.